other/materials_designer/Introduction.ipynb

```diff
@@ -0,0 +1,32 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Materials Transformations with Python Notebooks\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This Jupyter Lite distribution contains pre-configured examples of materials transformations including simple ones like creation of supercell or a surface, and more complex like creating an interface with Zur and McGill Superlattice matching algorithm\n",
+    "\n",
+    "## Examples\n",
+    "\n",
+    "### Zur and McGill Superlattice interface creation\n",
+    "\n",
+    "[This file](zsl_interface_generation.ipynb) helps creating interface between two materials by finding matching superlattices and selecting optimal variants by size-strain trade off.\n",
```

Comment:
Should we cite Zur McGill paper here? [Lattice match: An application to heteroepitaxy](https://pubs.aip.org/aip/jap/article-abstract/55/2/378/502944/Lattice-match-An-application-to-heteroepitaxy?redirectedFrom=fulltext)

Reply:
Added DOI link